pkgs/development/python-modules/atomman/default.nix

```diff
@@ -21,19 +22,27 @@
 }:
 
 buildPythonPackage rec {
-  version = "1.4.4";
+  version = "1.4.5";
   pname = "atomman";
   format = "setuptools";
 
-  disabled = pythonOlder "3.6" || pythonAtLeast "3.10";
+  disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "usnistgov";
     repo = "atomman";
     rev = "v${version}";
-    hash = "sha256-iLAB0KMtrTCyGpx+81QfHDPVDhq8OA6CDL/ipVRpyo0=";
+    hash = "sha256-wXz/uHjXKHVKJu/HoFF2mADSBLp6UGF9ivOp2ZOz/Ys=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "fix-tests.patch";
+      url = "https://github.com/usnistgov/atomman/pull/9.patch";
```

Comment:
Fetching entire unmerged PRs is forbidden due to reproducibility. Please fetch each commit.

Reply:
Good catch. Fixed.